### DISCUSSION TITLE
About food effect on PBPK modeling

### DISCUSSION THREAD

**Discussion by Harris728** [Open-Systems-Pharmacology/Forum#1208] [Questions & Problems]
Hi all,

I have a question about the food effect on PBPK modeling. Which parameters (physiological and anatomic) changed when I defined a special event in the Event Building Block? And which other parameters need me to adjust manually? I didn't find relevant descriptions in PK-Sim Documentation.

Could you please help me with this question?

Thanks in advance.
Harris

**Comment by AndreDlm** [comment]
Dear Harris,

it's described in the Manual [here](https://docs.open-systems-pharmacology.org/working-with-pk-sim/pk-sim-documentation/pk-sim-events#definition-of-new-events-in-pk-sim-r):
![image](https://user-images.githubusercontent.com/26541603/165758249-4bb60ae2-0390-4bad-ab22-a7026b82d451.png)

Also have a look at the [previous discussions on food effect](https://github.com/Open-Systems-Pharmacology/Forum/discussions?discussions_q=food+effect) where many features and additional aspects are described.

André

**Reply by Harris728** [reply]
Dear André, 

Thank you that you have taken the time to respond.
I will check more details in previous discussions.

Best, 
Harris
